src/Main.elm

```diff
@@ -2,45 +2,57 @@ module Main exposing (..)
 
 import Http
```

Comment:
Nit: this file is pretty large, probably need to disperse some code.

Reply:
That is true. I will work on this for the next PR.